openmicroscopy/openmicroscopy
# Get thumbnails (rebased onto develop)

manual rebase of https://github.com/openmicroscopy/openmicroscopy/pull/5062

--------------

# What this PR does

- allow loading thumbnails in a batches

# Testing this PR

- test thumbnail version like in https://github.com/openmicroscopy/openmicroscopy/pull/3713#issuecomment-95146309
- load plate, all thumbnail should be loaded in a batches
- log in and test if updating rendeting update thumbnail:
  - test centre panel, search and plate grid
- consider any usecases I could miss! :)

--depends-on #5207
